packages/myst-to-typst/tests/cross-references.yml

```diff
@@ -35,4 +35,4 @@ cases:
             - type: text
               value: 'a for the first part of the figure!'
     typst: |-
-      See #[@fig1]a for the first part of the figure!
+      See #link(<fig1>)[]a for the first part of the figure!
```

Comment:
I think that this actually is the case where we want this to continue happening with an `@id` now.

Right now this is empty link text.

Reply:
Nice, yeah, I was thinking about this - wasn't sure if we wanted MyST default text or typst default text. Thanks for fixing it up!